roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -31,6 +31,20 @@
   failed_when: false
   when: openshift.common.is_containerized | bool
 
+- name: Pre-pull node image
+  command: >
```

Comment:
Can we leverage the ansible docker role?

http://docs.ansible.com/ansible/docker_module.html